### PR TITLE
Calculate and generate fixtures for limits

### PR DESCRIPTION
**Pull request by catalinjitea**

### CODE REVIEW

File: ozone/core/management/commands/mkfixtures_limits.py [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+    PartyHistory
+)
+
+
+class Command(BaseCommand):
+    help = "Calculate Limits and generate fixtures."
+
+    OUTPUT_DIR = settings.FIXTURE_DIRS[0]
+
+    def handle(self, *args, **options):
```
Comment (sorinstelian) [inline ozone/core/management/commands/mkfixtures_limits.py]: Maybe it should exit with an error if no control measures exist.

File: ozone/core/management/commands/mkfixtures_limits.py [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+                        ).first()
+                        if not baseline or baseline.baseline is None:
+                            continue
+                        limit = baseline.baseline * cm.allowed
+                        data.append(
+                            self.get_entry(idx, party, period, group, limit_type.value, limit)
+                        )
+                        idx += 1
+                    elif length == 2:
+                        cm1 = cm_queryset_by_limit_type[0]
```
Comment (sorinstelian) [inline ozone/core/management/commands/mkfixtures_limits.py]: I think the queryset should be ordered for this to work properly.

File: ozone/core/management/commands/mkfixtures_limits.py (hunk 1 above)
Comment (catalinjitea) [reply ozone/core/management/commands/mkfixtures_limits.py]: Done.

File: ozone/core/management/commands/mkfixtures_limits.py (hunk 2 above)
Comment (catalinjitea) [reply ozone/core/management/commands/mkfixtures_limits.py]: Done, but fixture file hasn't changed.

**Comment by melish** [comment]
Why different rounding rule for BDN_NA5 and NA5_Prod baselines?

image

**Pull request merged**